### PR TITLE
Release

### PR DESCRIPTION
**Pull request by avi-biton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -382,6 +382,129 @@ should see another pipeline run called `test-component-c6glg-...` being triggere
...
+### Configure Release
+
+You will now configure Konflux to release your application to the external registry
+configured in previous steps.
+
+This requires:
+
+* A pipeline that will run on push events to the component repository.
+
+* ReleasePlan and ReleasePlanAdmission resources, that will react on the snapshot to
```
Comment (avi-biton) [inline README.md]:
```suggestion
* `ReleasePlan` and `ReleasePlanAdmission` resources, that will react on the snapshot to
```

File: README.md [hunk 2]
```diff
@@ -382,6 +382,129 @@ should see another pipeline run called `test-component-c6glg-...` being triggere
...
+  be created after the on-push pipeline will be triggered, which, in turn, will trigger
+  the creation of the release.
+
+#### Create the on-push Pipeline
+
+You will now introduce a new pipeline that will be triggered whenever new commits are
+created on branch `main` (e.g. when PRs are merged).
+
+On your local clone to the testrepo repository, copy the pipeline to the `.tekton`
```
Comment (avi-biton) [inline README.md]:
```suggestion
On your local clone of the testrepo repository, copy the pipeline to the `.tekton`
```

File: README.md (hunk 1 above)
Comment (yftacherzog) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (yftacherzog) [reply README.md]: done

**Review by gbenhaim — APPROVED** [review]

**Review by yftacherzog — APPROVED** [review]
@avi-biton are you ok with merging?

**Pull request merged**